_finite is obsoleted and C99 has isfinite instead so we should use it

Signed-off-by: Khem Raj <raj.khem@gmail.com>

Upstream-Status: Pending

Index: clutter-box2d-0.10.0/box2d/Source/Common/b2Math.h
===================================================================
--- clutter-box2d-0.10.0.orig/box2d/Source/Common/b2Math.h	2008-12-16 06:42:10.000000000 -0800
+++ clutter-box2d-0.10.0/box2d/Source/Common/b2Math.h	2011-07-17 02:10:29.833825198 -0700
@@ -60,7 +60,7 @@
 #ifdef _MSC_VER
 	return _finite(x) != 0;
 #else
-	return finite(x) != 0;
+	return std::isfinite(x) != 0;
 #endif
 }
 
